apps/trading/specs/index.spec.tsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
 
 import Index from '../pages/index.page';
 import { VegaWalletContext } from '@vegaprotocol/wallet';
 
 describe('Index', () => {
-  it('should render successfully', () => {
-    const { baseElement } = render(
-      <VegaWalletContext.Provider
-        value={{
-          keypair: null,
-          keypairs: null,
-          connect: jest.fn(),
-          disconnect: jest.fn(),
-          selectPublicKey: jest.fn(),
-          connector: null,
-          sendTx: jest.fn(),
-        }}
-      >
-        <Index />
-      </VegaWalletContext.Provider>
-    );
-    expect(baseElement).toBeTruthy();
+  it('should render successfully', async () => {
```

Comment:
chore (blocking): We can remove the mocked VegaWalletContext.Provider wrapping component, its not required for the test anymore